spec/system/tasks_spec.rb

```diff
@@ -0,0 +1,70 @@
+require 'rails_helper'
+
+RSpec.describe "Tasks", type: :system do
+
+  describe 'ログイン前' do
+    context 'タスク作成するとき' do
+      it 'タスク作成が失敗' do
+      end
+    end
+    context 'タスク編集するとき' do
+      it 'タスク編集が失敗' do
+      end
+    end
+  end
+  
+  describe 'ログイン後' do
+    describe 'タスク作成' do
+      context '入力値が正常のとき' do
+        it 'タスク作成が成功' do
+        end
+        it 'フラッシュメッセージが表示' do
+        end
+        it '入力値が画面に表示されていること' do
+        end
```

Comment:
itのテストケースの中の確認観点として記載した方が良いですね。
contextの条件で「タスク作成が成功する」というテストケースで、どんな確認項目があるか、といった感じで考えていくと良さそうです。
```suggestion
        it 'タスク作成が成功' do
          expect 'フラッシュメッセージが表示されること' 
          expect  '入力値が画面に表示されていること' 
        end
```